liwords-ui/src/gameroom/player_cards.tsx

```diff
@@ -48,7 +49,8 @@ const PlayerCard = React.memo((props: CardProps) => {
   const timeStr =
     isExamining || props.playing ? millisToTimeStr(props.time) : '--:--';
   // TODO: what we consider low time likely be set somewhere and not a magic number
-  const timeLow = props.time <= 180000 && props.time > 0;
+  const timeLowCutoff = Math.max(props.initialTimeSeconds / 5, 30000);
```

Comment:
How were these arrived at? It seems that having a 3 minute timer would result in for example an arbitrary-looking cutoff of 36 seconds, 4 minutes would be 48 seconds, etc. It's a bit strange. I think if there was some rounding to the nearest 15 or 30 seconds it might be better.